extensions/Cake.Deploy.Variables.yml

```diff
@@ -4,7 +4,7 @@ NuGet: Cake.Deploy.Variables
 Assemblies:
 - /**/Cake.Deploy.Variables.dll
 Repository: https://github.com/ObjectivityLtd/Cake.Deploy.Variables.git
-ProjectUrl: https://github.com/ObjectivityLtd/Cake.Deploy.Variables/
+ProjectUrl: https://www.nuget.org/packages/Cake.Deploy.Variables/
```

Comment:
@Jericho It seems we might have a bug with AddinDisco. It's changing the project URL to the NuGet org link, for an old package that hasn't changed since 2019

Reply:
The reason why the GitHub URL was replaced by the NuGet URL is because the github url is no longer valid as evidenced by:

![image](https://github.com/cake-build/website/assets/112710/2e921919-cefa-408e-83dd-18794a48bacf)